e2e/__tests__/console.test.ts

```diff
@@ -59,3 +59,22 @@ test('the jsdom console is the same as the test console', () => {
   expect(wrap(rest)).toMatchSnapshot();
   expect(wrap(summary)).toMatchSnapshot();
 });
+
+test('does not error out when using winston', () => {
+  const {stderr, stdout, status} = runJest('console-winston');
+  const {summary, rest} = extractSummary(stderr);
+
+  expect(wrap(stdout)).toMatchSnapshot();
+  expect(wrap(summary)).toMatchSnapshot();
+  expect(wrap(rest)).toMatchInlineSnapshot(`
+PASS __tests__/console.test.js
```

Comment:
Using an inline snapshot here so that it's easier to visually see the error occuring for now